packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -541,6 +549,25 @@ const FormModal = () => {
         return;
         // Add/edit a field to a content type
         // Add/edit a field to a created component (the end modal is not step 2)
+      } else if (isCreatingCustomFieldAttribute) {
+        addCustomFieldAttribute(
+          { ...modifiedData, customField: customFieldUid },
+          forTarget,
+          targetUid,
+          actionType === 'edit',
+          initialData
+        );
```

Comment:
Now I see the usage, I'd update my comment from above the recommend to split these up. It would also simplify the signature:

```
        if (actionType === 'edit') {
          editCustomFieldAttribute({ });
        } else {
           addCustomFieldAttribute({ });
        }
        
```